### PR TITLE
implements #23 for html5

### PR DESCRIPTION
**Pull request by SmokyBob**
Implements #23

### CODE REVIEW

File: public/js/controllers/generalStatisticsForGooglersCtrl.js [hunk 1]
```diff
@@ -435,6 +435,11 @@ GdeTrackingApp.controller("generalStatisticsForGooglersCtrl",	function($rootScop
               for (var j=0;j<$scope.data.items[i].product_groups.length;j++)
               {
                 var product = $scope.data.items[i].product_groups[j];
+                //MSO - 2015-07-07 - Aggregate android tags
```
Comment (Scarygami) [inline public/js/controllers/generalStatisticsForGooglersCtrl.js]: No need for this comment :)

File: public/js/controllers/generalStatisticsForGooglersCtrl.js [hunk 2]
```diff
@@ -435,6 +435,11 @@ GdeTrackingApp.controller("generalStatisticsForGooglersCtrl",	function($rootScop
...
+                if (product =='#diagnostics' || product =='#appindexing' || product =='#deeplinking'){
+                  product = '#android';
+                  debugger;
```
Comment (Scarygami) [inline public/js/controllers/generalStatisticsForGooglersCtrl.js]: Should remove this.

File: public/js/controllers/generalStatisticsForGooglersCtrl.js [hunk 3]
```diff
@@ -444,7 +449,7 @@ GdeTrackingApp.controller("generalStatisticsForGooglersCtrl",	function($rootScop
                     return false;
                   }
                 });
-
+
```
Comment (Scarygami) [inline public/js/controllers/generalStatisticsForGooglersCtrl.js]: Being picky now, but there's some unnecessary white space here.

**Comment by Scarygami** [comment]
Good for a quick fix for now, but this should be handled by the (new) backend in the future.

**Comment by SmokyBob** [comment]
@Scarygami yep, just a quick fix for the current version.

> done by the backend

you mean adding a column like "chart_pg" to the single activity?

**Comment by Scarygami** [comment]
Something like this should probably be part of the productGroup entity, yes.

Btw, I noticed there's also an [#androidm tag](https://omega-keep-406.appspot.com/_ah/api/gdetracking/v1.0b2/productGroup/androidm) that isn't handled by your fix yet.

**Comment by SmokyBob** [comment]
... for some reason #androidm is not to be aggregate under #android ... at least yet/from the docs...
I'll shoot a mail to Natalie to see if they want to keep them separated for now or not.

**Comment by SmokyBob** [comment]
confirmed #androidm need to be aggregated too, so patched that in :)

File: public/bower.json [hunk 4]
```diff
@@ -18,5 +18,8 @@
     "paper-tabs": "Polymer/paper-tabs#~0.5.5",
     "paper-spinner": "Polymer/paper-spinner#~0.5.5",
     "webcomponentsjs": "WebComponents/webcomponentsjs#~0.5.5"
+  },
+  "resolutions": {
+    "webcomponentsjs": "^0.6.0"
```
Comment (Scarygami) [inline public/bower.json:23]: Can we get rid of this resolution by depending on webcomponentsjs^0.6.0 instead of ~0.5.5 above?

**Pull request merged**